source/pip/tests/test_qsharp.py

```diff
@@ -306,6 +306,17 @@ def test_run_with_noise_produces_noisy_results() -> None:
     assert result[0] > 5
 
 
+def test_run_with_loss_produces_lossy_results() -> None:
+    qsharp.init()
+    qsharp.set_quantum_seed(0)
+    result = qsharp.run(
+        "{ use q = Qubit(); X(q); MResetZ(q) }",
+        shots=1,
+        qubit_loss=1.0,
+    )
+    assert result[0] == qsharp.Result.Loss
+
+
```

Comment:
Does it make sense to add a test for comparing a lost qubit result to make sure we surface a nice error in Python?

Reply:
I could, but since it's a runtime error like any other I don't think it needs special callout here. We verify that it behaves like other runtime errors in our Rust unit tests.